ctapipe/io/containers.py

```diff
@@ -381,6 +396,8 @@ class TelescopePointingContainer(Container):
     """
     azimuth = Field(nan * u.rad, 'Azimuth, measured N->E', unit=u.rad)
     altitude = Field(nan * u.rad, 'Altitude', unit=u.rad)
+    ra = Field(nan * u.rad, 'Right ascension of camera center pointing from raw data', unit=u.rad)
```

Comment:
I don't think we should duplicate information here. Shouldn't this be handled by coordinate transforms?

Reply:
Some telescopes (e.g. MAGIC) provide this information right away. Can remove it if you insist.

Reply:
FACT also handles pointing in equatorial coordinates and transforms to horizontal as needed.
So it might make sense to keep all four and fill all of them from the data available.